rules/test_host_app/Info.plist

```diff
@@ -16,8 +16,6 @@
     <string>????</string>
     <key>CFBundleVersion</key>
     <string>1.0.0</string>
-    <key>NSPrincipalClass</key>
```

Comment:
Do you need to remove this both from `resource_bundle.plist` and the default test host app?

Reply:
Good point, let me try without this change and report back. Although I wonder if providing less is better here if it works as expected without the key? It doesn't look like we have it defined either unless we inject it somewhere in the rules.

Reply:
So looks like only the `test_host_app` info plist needs the change. Changing resources_bundle.plist doesn't have an effect here so I reverted that change.

Reply:
Ok let's run this change on CI and if it passes we can fix fallout as a followup

Reply:
Looks like it passed 🚀